static/css/daccs-css.css

```diff
@@ -168,13 +168,97 @@
     height: 25%;
     font-family: "Helvetica", sans-serif;
     color: #FFFFFF;
-    background-color: #000000;
+    background-color: rgba(0.0, 0.0, 0.0, 0.5);
+}
+
+#home-video{
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+    z-index: -100;
 }
 
 .banner-spacer{
     height: 20%;
 }
 
+.banner-video {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+    z-index: -100;
+}
+
+.banner-video-overlay{
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0.0, 0.0, 0.0, 0.25);
+}
+
+.section-container {
+    min-height: 100vh;
+    padding: 60px 0 0 0;
+    display: flex;
+    align-items: center;
+}
+
+.page-section {
+    position: relative;
+}
+
+.text-white {
+    color: #fff !important;
+}
+
+.bg{
```

Comment:
unused?

Reply:
"bg" class put back in for the static image that is used for smaller screen sizes.